cypress/e2e/mytest.spec.cy.js

```diff
@@ -0,0 +1,44 @@
+describe('Order Product', () => {
```

Comment:
This spec is not relevant to test task

Reply:
This is my own test as stated in the test task